cdap-docs/developers-manual/source/building-blocks/spark-programs.rst

```diff
@@ -98,6 +98,10 @@ Spark programs in CDAP can directly access **Dataset** similar to the way a MapR
 Procedure can. These programs can create Spark's Resilient Distributed Dataset (RDD) by
 reading a Dataset and can also write RDD to a Dataset.
 
+In order to access a Dataset in Spark, both the key and value classes has to be serializable.
```

Comment:
sorry, overlooked this first time. It is plural so it has to be ...have to be serializable.